main/zoe/api/zoe.md

```diff
@@ -193,11 +194,27 @@ be different for different instances of the same contract, but the contract
 defines what variables need their values passed in as `terms`.
 
 It returns a promise for a `StartInstanceResult` object. The object consists of:
+- `adminFacet` `{any}`
 - `creatorFacet` `{any}`
 - `publicFacet` `{any}`
 - `instance` `{Instance}`
 - `creatorInvitation` `{Payment | undefined}`
 
+The `adminFacet` has two methods:
+- `getVatShutdownPromise()
+  - Returns a promise that resolves (to reason or completion) when this newly started instance terminates. **tyg todo: Not sure what "resolves to reason" does? Thrown error?**
```

Comment:
'reason' is the conventional name for the Error generated with `fail(reason)`, while `completion` is the name passed with `exit(completion)`.